app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,29 @@
+class SessionsController < ApplicationController
+  def front
+    redirect_to videos_path if sign_in?
+  end
+
+  def new
+    redirect_to videos_path if sign_in?
+  end
+
+  def create
+    user = User.find_by_email(params[:email])
```

Comment:
I read two methods' source code,
find that  find_by_email is dynamic method and find_by calling where method. That is the reason?